dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/webcontainerext/JSPExtensionFactory.java

```diff
@@ -608,29 +575,15 @@ public static synchronized GlobalTagLibraryCache getGlobalTagLibraryCache() {
      */
     @Reference(cardinality=ReferenceCardinality.MULTIPLE, policy=ReferencePolicy.DYNAMIC)
     protected void setGlobalTagLibConfig(GlobalTagLibConfig globalTagLibConfig) {
-            
-            /*  
-                Cache the globalTagLibConfig argument until setVersion is called. 
-
-                setGlobalTagLibConfig creates the GlobalTagLibraryCache if it doesn't exist yet 
-                isPages30orHigher is called with in the GlobalTagLibraryCache constuctor.  
-                setGlobalTagLibConfig is called before setVersion and it created a problem 
-                since the JSP/Pages version is needed to determine which JSTL/Tags TLDs to load. 
-            */  
-            
-            if(!isPagesVersionLoaded){
-                storedGlobalTagLibConfigs.add(globalTagLibConfig);
-            } else {
```

Comment:
Just so I'm clear we're removing this because now that we're using the property files we don't need to worry about this timing issue any longer?

Reply:
Correct. No more timing issues.